### PR TITLE
Add isPlatformSupported()

### LINKED ISSUE #27: Add the ability to check if the platform is supported
Thanks for your awesome library!
I have a suggestion. Since hotkeys for Linux work only for X11, then could you add a function that checks if the platform is supported? This can be convenient if I want to show the user in the UI that hotkeys cannot be registered.
This can be implemented by adding a function that returns true for Windows and Mac, and [QX11Info::isPlatformX11()](https://doc.qt.io/qt-5/qx11info.html#isPlatformX11) on Linux.
I can make a PR if you approve my suggestion.

### PR DESCRIPTION
**Pull request by Shatur95**
closes #27

### CODE REVIEW

File: QHotkey/qhotkey_mac.cpp [hunk 1]
```diff
@@ -23,15 +23,19 @@ class QHotkeyPrivateMac : public QHotkeyPrivate
 	static QHash<QHotkey::NativeShortcut, EventHotKeyRef> hotkeyRefs;
 };
 NATIVE_INSTANCE(QHotkeyPrivateMac)
+bool QHotkeyPrivate::isPlatformSupported()
```
Comment (Skycoder42) [inline QHotkey/qhotkey_mac.cpp]: Please add a newline before the method

File: QHotkey/qhotkey_win.cpp [hunk 2]
```diff
@@ -22,11 +22,15 @@ class QHotkeyPrivateWin : public QHotkeyPrivate
 	static QString formatWinError(DWORD winError);
 };
 NATIVE_INSTANCE(QHotkeyPrivateWin)
+bool QHotkeyPrivate::isPlatformSupported()
```
Comment (Skycoder42) [inline QHotkey/qhotkey_win.cpp]: newline here too

File: QHotkey/qhotkey_x11.cpp [hunk 3]
```diff
@@ -46,14 +46,18 @@ class QHotkeyPrivateX11 : public QHotkeyPrivate
 	};
 };
 NATIVE_INSTANCE(QHotkeyPrivateX11)
+bool QHotkeyPrivate::isPlatformSupported()
```
Comment (Skycoder42) [inline QHotkey/qhotkey_x11.cpp]: ... and here as well

**Comment by Shatur95** [comment]
Added.

**Comment by Shatur95** [comment]
Thanks! Could you draft it as a new release?

**Pull request merged**